Simplified/SignInLogic/OELoginFirstBookVC.swift

```diff
@@ -141,6 +141,28 @@ class OELoginFirstBookVC: UIViewController {
     businessLogic.logIn()
   }
 
+  @IBAction func showTroubleshooting() {
+    let url = URL(string: "https://www.openebooks.org/signin-help")!
```

Comment:
Is there a reason we don't store this and the URLs below in the `NYPLLibraryAccountURLsProvider` class?

Reply:
yeah, I looked at that and that logic is hooked up with the Account class and Settings logic, so it didn't seem the right thing to do. These links are not included in the authentication document or library catalog after all.